### PR TITLE
Add support for running perffuzz in infra/helper.py

### PR DESCRIPTION
**Pull request by inferno-chromium**
https://github.com/carolemieux/perffuzz

### CODE REVIEW

**Comment by inferno-chromium** [comment]
This is just to support local testing, there is no current plan to integrate this in ClusterFuzz. I really like the concept of maximum execution counts for all edges (still worrying about corpus explosion, similar to value profiling). @kcc as fyi.

File: infra/base-images/base-runner/run_fuzzer [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eu
-# Copyright 2016 Google Inc.
+# Copyright 2018 Google Inc.
```
Comment (Dor1s) [inline infra/base-images/base-runner/run_fuzzer]: nit: I guess you intended to mean the year in compile_perffuzz file

Comment (jonathanmetzman) [reply infra/base-images/base-runner/run_fuzzer]: +1

Comment (inferno-chromium) [reply infra/base-images/base-runner/run_fuzzer]: Yes Sir!, you are right!

File: infra/helper.py [hunk 2]
```diff
@@ -240,7 +240,8 @@ def _get_work_dir(project_name=''):
 def _add_engine_args(parser):
   """Add common engine args."""
   parser.add_argument('--engine', default='libfuzzer',
-                      choices=['libfuzzer', 'afl', 'honggfuzz', 'none'])
+                      choices=['libfuzzer', 'afl', 'honggfuzz', 'perffuzz',
```
Comment (jonathanmetzman) [inline infra/helper.py]: Can we document somewhere (preferably the help message) that this is fuzzer is an academic experiment that we are playing around with and that this unsupported.

Comment (inferno-chromium) [reply infra/helper.py]: I wonder where we can add this ? in infra/helper.py build_fuzzers and run_fuzzer or you were thinking docs ? putting in docs i feel makes more confusing ?

Comment (jonathanmetzman) [reply infra/helper.py]: Maybe just a comment somewhere then?

**Pull request merged**